integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/model/DatabaseUtils.groovy

```diff
@@ -31,6 +31,11 @@ class DatabaseUtils {
     sql.executeUpdate("insert into form values (-1, ?::JSON, $bookingId, 'CATEGORISER_USER', 'STARTED', 'CATEGORISER_USER')", json)
   }
 
+  def createDataWithStatus(bookingId, status, json) {
+    def sql = Sql.newInstance(dbConnParams)
```

Comment:
$status in single quotes .. does that work?

Reply:
it was the only way i could get it to work -  looks fine on the db etc